_editorial_policies/03_reviewer_information.md

```diff
@@ -0,0 +1,111 @@
+
+Thank you very much.
+`{{SENDER-FULL-NAME}}`, Associate Editor
+
+`{{MANUSCRIPT-TITLE}}`
+Abstract:
+(abstract will be automatically included)
+
+
+### Sample review action/thanks for agreeing letter
+
+(Each category has its own form letter, differing in the link to category-specific review criteria. The below sample is for Best Practices Guides)
+
+Dear XYZ,
+
+Thank you very much for agreeing to review the LiveCoMS submission, `{{MANUSCRIPT-TITLE}}`.
+Your support of our new community-run journal is very encouraging.
+
+I would appreciate receiving your review within 10 days, by `TTTTTT`.
+
+The article is to be considered under the `YYY` category.
+To aid you in your review, both the general [review criteria](https://livecomsjournal.github.io/authors/policies/#review-criteria) and the criteria specific to [Best Practices Guides](https://livecomsjournal.github.io/authors/best_practices/) are appended below.
+
+Please read our conflict-of-interest criteria and inform me of any potential conflicts.
+
+Note also that non-anonymous feedback can be provided directly to authors via the GitHub page linked in the manuscript, if you so desire.
+
+If you have any questions, please reply to this email.
+
+Thank you very much.
+`{{SENDER-FULL-NAME}}`, Associate Editor
+
+REVIEW CRITERIA
+`Fill in current general review criteria`
+`Fill in current category-specific review criteria`
+
+### Reviewer ranking questions
+
+(All fields will be communicated to the authors except comments to the editor)
+- Overall rating (one to five stars)
+- For this manuscript I recommend (Accept/Revise and resubmit/Reject)
+- How well does the manuscript meet overall review criteria and category-specific criteria?
+- Is the paper adequately edited?
+- Given that the accepted PDF will be the final version of the manuscript, are there formatting, style, or quality issues which should be addressed before acceptance? What about the quality of the figures, layout of any tables, etc.?
+- Comments to the editor
+
+
+## Sample decision letter
+
+Following is a sample decision letter which might be used for communicating the outcome of the review process to the authors.
+
+Dear `{{AUTHOR-FULL-NAME}}`,
+
+Thank you very much for your submission of your article, `{{MANUSCRIPT-TITLE}}`, to the Living Journal of Computational Molecular Sciences.
+We are delighted to inform you that we anticipate being able to accept your article, pending satisfactory completion of minor revisions as recommended by the reviewers (and detailed below).
+As you make the recommended changes, we also encourage you to make a final check that:
+- The article is well edited and all images and tables are of high quality and well laid out
+- Your final submitted PDF is in fact how you want your article to appear in its final version, as the journal does not typeset or alter the formatting of your article
+- Your GitHub repository is in order and ready to accept feedback from readers
+
```

Comment:
I think highlighting is a big help in looking at a revised manuscript.  I suggest:
"To facilitate the re-review of your manuscript, please submit a pdf version with *highlighted* changes.  A final unhighlighted, publication-ready version will be requested should your manuscript be accepted."

Reply:
Good idea. Done.